docs/source/cli_help/file.rst

```diff
@@ -10,3 +10,11 @@ file
 
 Outputs the absolute path of the machete definition file. Currently fixed to ``<git-directory>/machete``.
 Note: this won't always be just ``<repo-root>/.git/machete`` since e.g. submodules and worktrees have their git directories in different location.
+
+Outputs the absolute path of the machete definition file.
```

Comment:
`Outputs the absolute path of the machete definition file.` doubled?

Reply:
Whoops good spot